ios/Runner/Info.plist

```diff
@@ -22,7 +22,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(FLUTTER_BUILD_NUMBER)</string>
+	<string>4</string>
```

Comment:
@ryan-conway I'm not sure if we should revert this change? Is there `FLUTTER_BUILD_NUMBER` defined in somewhere?